File: Make Matrix Beautiful.java
class Solution {
    public static int findMinOperation(int N, int[][] matrix) {
    // code here
      int max =0;
      int [] row = new int [N];
      int [] col = new int [N];
      
      for (int i =0; i< N; i++){
          for (int j =0; j<N; j++){
              row[i]+=matrix[i][j];
              col[i]+=matrix[j][i];
          }
          max = Math.max(Math.max(row[i], col[i]), max);
      }
      int ans =0;
      for (int i =0; i< N ;i++){
          ans+=max-row[i];
      }
      return ans;
      
    }
}
        

File: Make Matrix Beautiful.cpp
 public:
    //Function to find minimum number of operations that are required 
    //to make the matrix beautiful.
    int findMinOpeartion(vector<vector<int> > matrix, int n)
    {
        // code here 
        int total = 0;
        for(int i=0;i<n;i++){
            int sum1 = 0;
            int sum2 = 0;
            
            for(int j=0;j<n;j++){
                sum1 = sum1 + matrix[i][j];
                sum2 = sum2 + matrix[j][i];
            }
            total = max(total,max(sum1,sum2));
        }
        
        int cnt = 0;
        for(int i=0;i<n;i++){
            int sum = 0;
            
            for(int j=0;j<n;j++){
                sum = sum + matrix[i][j];
            }
            cnt += total -sum;
        }
        
        return cnt;
    } 
